lbuild/main.py

```diff
@@ -543,7 +543,7 @@ def main():
 
     except lbuild.exception.LbuildException as error:
         sys.stderr.write('\nERROR: {}\n'.format(error))
-        if args.verbose >= 2:
+        if args.verbose >= 1:
             traceback.print_exc()
```

Comment:
I think that in case of an error the stacktrace should always be printed. E.g. in the CI it is very time consuming to run lbuild again with `-v`.

```suggestion
        traceback.print_exc()
```

Reply:
lbuild uses exceptions for all errors, and leaves it up to them to format the error correctly. It would be very confusing to print a stack trace for all these errors. LbuildForwardException is used to catch errors in user code and prints the stack trace itself.

Reply:
For reference, these are all the other exceptions that can also occur: https://github.com/modm-io/lbuild/blob/develop/lbuild/exception.py